OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+    Status: Downloaded newer image for container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+    container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+	
+  8. Run the `docker tag` command to tag the image as follows:
+  
+    docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4 oracle/fmw-infrastructure:12.2.1.4.0
+	
+   No output is returned to the screen.
+
+  9. Run the `docker images` command to show the image is installed into the repository. The output should look similar to this:
+	
+	$ docker images
+	
+    REPOSITORY                                                    TAG                 IMAGE ID            CREATED             SIZE
+    container-registry.oracle.com/middleware/fmw-infrastructure   12.2.1.4            760157811a73        4 weeks ago         2.33GB
+    oracle/fmw-infrastructure                                     12.2.1.4.0          760157811a73        4 weeks ago         2.33GB
+    container-registry.oracle.com/java/serverjre                  8                   da8a8a240247        2 months ago        305MB
+    oracle/serverjre                                              8                   da8a8a240247        2 months ago        305MB
+	
+
+	
+### Downloading the OAM Docker files
+
+  1. Make a work directory to place the OAM Docker files:
+     
+	$ mkdir <work directory>
+	
+  2. Download the OAM Docker files from the OAM [repository](https://github.com/oracle/docker-images) by running the following command:
+
+   
+    $ cd <work directory>
+	$ git clone https://github.com/oracle/docker-images
+
+### Downloading the 12.2.1.4.0 Identity Management shiphome and Patches
+
+  1. Download the [Oracle Identity and Access Management 12cPS4 software](https://www.oracle.com/middleware/technologies/identity-management/downloads.html) to a stage directory. Unzip the downloaded `fmw_12.2.1.4.0_idm_Disk1_1of1.zip` file and copy the `fmw_12.2.1.4.0_idm.jar` to `<work directory>/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/`:
+    
+	$ unzip fmw_12.2.1.4.0_idm_Disk1_1of1.zip
+    $ cp fmw_12.2.1.4.0_idm.jar <work directory>/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/fmw_12.2.1.4.0_idm_generic.jar
+	
+   **Note**: The filename must be changed to `fmw_12.2.1.4.0_idm_generic.jar` when copying to the 12.2.1.4.0 directory.	
+   
+  2. Create the following directories under the `12.2.1.4.0` directory:
+	
+	$ mkdir -p <work directory>/OracleAccessManagement/dockerfiles/12.2.1.4.0/patches
+	$ mkdir -p <work directory>/OracleAccessManagement/dockerfiles/12.2.1.4.0/opatch_patch
+	
+  3. View the latest `manifest.<date>.properties` from this [repository](./imagetool/12.2.1.4.0).	
+  
+     Look at the `[XXXX_PATCH]` sections and download the one off patches referenced. For example, the `manifest.oam.july2020.properties` below shows the following required patches under `[INFRA_PATCH]` and `[OAM_PATCH]`:     
```

Comment:
Can you get these patches if you're not a customer? If not, we cannot publish this build until they're available without requiring a CSI.